docs/application/dotnet/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,141 @@
+
+    ```
+    using Tizen.Applications.ComponentBased;
+    ```
+
+<a name="manage_context"></a>
+## Managing Running Component Context
+
+To get the component running context and its details, and to operate on the context:
+
+1.  Get the context of the currently running component by creating an instance of the [Tizen.Applications.ComponentBased.ComponentRunningContext](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentRunningContext.html) class, with the ID of the context obtained component as a parameter.
+
+    To get a component's context, the component must be running.
+
+    ```
+    ComponentRunningContext compoRunningContext = new ComponentRunningContext(Your Component ID);
+    ```
+
+2.  Operate on the context:
+    -   Get the component ID, application ID, and instance ID from the context:
+
+        ```
+        string componentId = compoRunningContext.ComponentId;
+        string applicationId = compoRunningContext.ApplicationId;
+        string instanceId = compoRunningContext.InstanceId;
+        ```
+
+    -   Check the state of the component:
+
+        ```
+        if (compoRunningContext.State == ComponentRunningContext.ComponentState.Initialized)
+            /// The component is constructed.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Created)
+            /// The component is created.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Started)
+            /// The component is started.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Resumed)
+            /// The component is resumed.
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Paused)
+            /// The component is paused
+        else if (compoRunningContext.State == ComponentRunningContext.ComponentState.Destroyed)
+            /// The component is destroyed.
+        else
+            /// State is undefined
+        ```
+
+    -   Resume the running application:
+
+        ```
+        compoRunningContext.Resume();
+        ```
+
+<a name="filter"></a>
+## Managing Component Information
+
+To get the installed information and its details, and to operate on the information:
+
+1.  Get the information of the currently-installed component by creating an instance of the [Tizen.Applications.ComponentBased.ComponentInfo](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentInfo.html) class, with the ID of information obtained component as a parameter.
+
+    To get a component's information, the component must be installed.
+
+    ```
+    ComponentInfo compoInfo = new ComponentInfo(Your Component ID);
+    ```
+
+2.  Operate on the context:
```

Comment:
Isn't "Operate on the information:" correct? Would you please check?

Reply:
@mijin85cho, yes, it should be: Operate on the information

Reply:
Done